.travis.yml

```diff
@@ -5,7 +5,12 @@ os:
   - osx
 julia:
   - 0.5
-  # - nightly
+  - 0.6
+  - nightly
+matrix:
+  allow_failures:
+    - julia: 0.6
```

Comment:
While we are at it, would you mind removing 0.6 from here -- I'd like to ensure that those tests are passing.